activestorage/test/service/azure_storage_service_test.rb

```diff
@@ -9,11 +9,13 @@ class ActiveStorage::Service::AzureStorageServiceTest < ActiveSupport::TestCase
 
     include ActiveStorage::Service::SharedServiceTests
 
-    test "upload with content_type" do
+    test "upload with content type and content disposition" do
```

Comment:
Can we have separate tests for specifying content types and content disposition?

Reply:
Done!